addons/sale/models/sale_order_line.py

```diff
@@ -926,6 +965,10 @@ def create(self, vals_list):
                 # create an analytic account if at least an expense product
                 if line.product_id.expense_policy not in [False, 'no'] and not line.order_id.analytic_account_id:
                     line.order_id._create_analytic_account()
+
+        # call the SO to create a section of Down Payments in case a down payment is created
+        if any(line.is_downpayment for line in lines):
+            lines.order_id._on_downpayment_creation()
```

Comment:
We do not want to support this here.
The down payment section creation should only be covered and managed in the invoicing wizard, no need to consider the case when custom code adds down payment lines.